### PR TITLE
Chat block user

### PR DESCRIPTION
**Pull request by rmisio**
Implemented functionality to block / unblock a user from the chat menu as well as keep the menu in sync as the related users are being blocked / unblocked (from the chat app or elsewhere).

### CODE REVIEW

File: js/views/chatAppVw.js [hunk 1]
```diff
@@ -140,17 +161,19 @@ module.exports = Backbone.View.extend({
   },
 
   renderNoneFound: function(){
-    // Decide what to do here
+    this.$('#chatHeads').html('<p>No results found.</p>');
```
Comment (jjeffryes) [inline js/views/chatAppVw.js]: Use a translation here

Comment (rmisio) [inline js/views/chatAppVw.js]: updated.

**Comment by jjeffryes** [comment]
This looks pretty good. One minor issue, if you block someone, then restart the client, then unblock them in the settings view, their chats don't reappear in the chat window until you change views a few times or restart the client again.

**Comment by jjeffryes** [comment]
One other minor thing, if you block someone from chat while you're in the settings block tab, they don't appear in the list of blocked users until you re-render that page.

**Comment by rmisio** [comment]
@jjeffryes The seconds thing I'm aware of. I actually have a branch where I'm working on a patch, but I've put it aside for now. I'll get back to it when I have some down-time. I'll also look into the first issue you mentioned.

**Pull request merged**